RMv4_Recipely/RMv4_Recipely/Authorization/AuthViewController.swift

```diff
@@ -3,41 +3,337 @@
 
 import UIKit
 
-protocol AuthViewControllerProtocol: AnyObject {}
-
 /// Экран авторизации
 final class AuthViewController: UIViewController {
     // MARK: - Constants
 
     private enum Constants {
-        //    static let
-        //    static let
-        //    static let
-        //    static let
+        static let loginText = "Login"
+        static let emailAddressText = "Email Address"
+        static let verdana = "Verdana"
+        static let verdanaBold = "Verdana-Bold"
+
+        static let passwordText = "Password"
+        static let incorrectFormatText = "Incorrect format"
+        static let wrongPasswordText = "You entered the wrong password"
+        static let emailAddressPlaceholderText = "Enter Email Address"
+        static let passwordPlaceholderText = "Enter Password"
+
+        static let loginButtonText = "Login"
+        static let errorMessageText = "Please check the accuracy of the\n entered credentials."
     }
 
     // MARK: - Visual Components
 
-    // MARK: - Public Properties
+    private lazy var loginLabel: UILabel = {
+        let element = UILabel()
```

Comment:
```suggestion
        let  label = UILabel()
```
лучше называть производным от класса